media-libs/libmatemixer/libmatemixer-1.26.1.ebuild

```diff
@@ -38,6 +38,15 @@ RDEPEND="${COMMON_DEPEND}
 DEPEND="${COMMON_DEPEND}
 "
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.26.1-slibtool.patch #921214
+)
+
+src_prepare() {
```

Comment:
mate.eclass provides its own:
```
       mate_src_prepare
              Call gnome2_src_prepare to handle environment setup and patching, then call eautoreconf if necessary
```

Please use that instead.

Reply:
Thanks, fixed.